recipes/runtimeqml/all/conanfile.py

```diff
@@ -41,28 +41,24 @@ def _compilers_minimum_version(self):
         }
 
     def export_sources(self):
-        copy(self, "CMakeLists.txt", self.recipe_folder,
-             self.export_sources_folder)
+        copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=self.export_sources_folder)
```

Comment:
```suggestion
        copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=os.path.join(self.export_sources_folder, "src"))
```
To avoid the extra copy() in build().